pkg/kubecost/totals.go

```diff
@@ -295,9 +295,9 @@ func ComputeAssetTotals(as *AssetSet, prop AssetProperty) map[string]*AssetTotal
 				key = fmt.Sprintf("%s/%s", node.Properties().Cluster, node.Properties().Name)
 			}
```

Comment:
Can we just remove the key variable and this conditional?

Reply:
No, we use it later on. It's just been replaced in that one line.

(Besides, Pike wouldn't let me keep it if I weren't using it :rofl:)